tests/compile-fail/stacked_borrows/track_id.rs

```diff
@@ -0,0 +1,19 @@
+// compile-flags: -Zmiri-track-id=1372
```

Comment:
This test will break any time *anything* in the libstd prelude changes. I don't think we should have to update Miri each time that happens.

I'd rather have no test than this test, TBH.

Reply:
I can work on making miri run no-std tests again, then there'll be nothing changing the ids. But yea, for now I'll remove the test